### DISCUSSION TITLE
How to save file from image input?

### DISCUSSION THREAD

**Discussion by erikpa1** [tauri-apps/tauri#5450] [Q&A] [answered]
Hi,

I have form, where I have image, which I want to save on disc after user press save. 

How to say tauri from where it should take file and copy to destination ? 

I dont need tauri save popup because I know where I'm  saving my data, I need just need path and permision to take that file and copy it

![image](https://user-images.githubusercontent.com/19526670/196917097-7ad9eeed-adc6-4b81-8152-153874434b18.png)

**Comment by FabianLars** [comment, marked as answer]
If you used the webview's built-in file dialog than you can only save the file by piping it's binary data through [writeBinaryFile](https://tauri.app/v1/api/js/fs/#writebinaryfile). If you used Tauri's own [open dialog](https://tauri.app/v1/api/js/dialog#open) you'll already have the path which you can use to copy the file.

**Reply by erikpa1** [reply]
@FabianLars thanks, I will try

**Reply by erikpa1** [reply]
@FabianLars Thanks, its working :)

**Comment by ghost** [comment]
Sorry for nontopic question. Is this bootstrap?
